plugins/references.py

```diff
@@ -86,6 +93,15 @@ def getMarkers(view, name=""):
     return ids
 
 
+def find_by_selector_in_regions(view, regions, selector):
+    def _gen():
+        for sel in view.find_by_selector(selector):
+            if any(s.intersects(sel) for s in regions):
+                yield sel
+
+    return list(_gen())
```

Comment:
If you return a list ony, why bothering with generators. That doesn't seem to make sense. Either the function is a generator or not.

Reply:
Yeah, this is just using the generator as syntactic sugar right now. A leftover from testing.